resources/lemmas.md

```diff
@@ -11,6 +11,16 @@ module LEMMAS
     imports K-REFLECTION
 ```
 
+### Gas Abstraction
+```k
+    syntax Int ::= "#infinity" [function]
+ // -------------------------------------
+    rule #infinity -Int _ => #infinity
+    rule #infinity >=Int _ => true
+    rule #infinity <Int _ => false
```

Comment:
could you put some comments here, saying that it is temporary?

Reply:
done